src/auth/storage.js

```diff
@@ -24,30 +24,35 @@ class MemoryStorage {
 }
 
 class Storage {
-  constructor(store) {
+  /**
+   * @param {Storage} store implementer of Storage interface
+   * @param {String} namespace appID to be used for namespacing
+  */
+  constructor(store, namespace) {
     this.store = store;
+    this.namespace = namespace;
   }
 
-  get(key) { return new Promise(resolve => resolve(this.store.getItem(key))); }
-  set(key, value) { return new Promise(resolve => resolve(this.store.setItem(key, value))); }
-  remove(key) { return new Promise(resolve => resolve(this.store.removeItem(key))); }
+  get(key) { return new Promise(resolve => resolve(this.store.getItem(`${this.namespace}.${key}`))); }
```

Comment:
[test] We should test that creating multiple apps works and that you can be logged in to two at the same time. As well as a test for logging out of one and being logged into the other.